### ISSUE TITLE
telescope.py: implement accelerating slew model class

### ISSUE THREAD

**Issue by skiehl** [skiehl/obsplanner#7]

**Comment by HeisenbergK** [comment]
Good!

Should I make a fork and work there, or make the changes in master and implement the obsplanner as a module to another repo which will cover the survey strategy.

I guess I'm asking, are we separating the obsplanner from "PASIPHAE_survey_planner" (which I think is better)?

If yes, I'll just push my changes in master.

**Comment by skiehl** [comment]
You can push directly to master.

Let's implement the survey planner in a dedicated repo, because it will be very strongly tied to Pasiphae, whereas this is going to be a broad purpose software.

**Comment by HeisenbergK** [comment]
Agreed. This way it's better. So this repo, plus 1 for the survey planner, which will include the wrapper
